server/integration/testsuite/src/test/java/org/infinispan/server/test/rollingupgrades/HotRodRollingUpgradesIT.java

```diff
@@ -33,30 +33,7 @@
  */
 @RunWith(Arquillian.class)
 @Category({RollingUpgrades.class})
```

Comment:
I'm going to implement testing against `old.server.schema.version` which should be set to previous major ISPN release but can be also easily changes from cmd line/profile to test older releases.